src/component/DetailCard.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Segment, Placeholder } from 'semantic-ui-react';
+import theme from '../config/theme';
+
+const DetailCard = ({
+  title,
+  location,
+  description,
+  price,
+  imageLink,
+  /* eslint-disable-next-line */
+} = {}) => {
+  return title === undefined ? (
```

Comment:
A neater way is to write **return !title**

Reply:
got it!